custom/icds_reports/templates/icds_reports/icds_app/cas-export.directive.html

```diff
@@ -25,16 +25,18 @@
                 </div>
             </div>
             <div class="form-group">
-                <label class="control-label col-sm-2" for="indicator">State: <i class="fa fa-info-circle" uib-popover="Select the location."
+                <label class="control-label col-sm-2" for="indicator">Location: <i class="fa fa-info-circle" uib-popover="Select the location."
                        popover-placement="top-right"
                        popover-trigger="'mouseenter'"></i></label>
                 <div class="col-sm-6">
-                    <ui-select ng-model="$ctrl.selectedLocation" theme="bootstrap">
-                        <ui-select-match placeholder="State"><span ng-bind="$select.selected.name"></span></ui-select-match>
-                        <ui-select-choices repeat="indicator.location_id as indicator in $ctrl.locations | propsFilter: {name: $select.search}">
-                            <div ng-bind-html="indicator.name"></div>
-                        </ui-select-choices>
+                  <div style="margin-bottom: 15px;" ng-repeat="locationTypes in $ctrl.hierarchy" ng-show="$ctrl.isLocationVisible($index)" ng-if="$index < 3">
```

Comment:
was this copied/modified from somewhere or is this new code?

Reply:
copied from https://github.com/dimagi/commcare-hq/blob/master/custom/icds_reports/templates/icds_reports/icds_app/download.directive.html#L91-L100

Reply:
roger. my dashboard duplication radar remains strong! :upside_down_face: 

could potentially add a todo / note indicating the duplication for a future reference, though realize html comments are a bit awkward